key.go

```diff
@@ -59,18 +60,13 @@ type PrivKey interface {
 
 	// Generate a secret string of bytes
 	GenSecret() []byte
-
-	Decrypt(b []byte) ([]byte, error)
 }
 
 type PubKey interface {
 	Key
 
 	// Verify that 'sig' is the signed hash of 'data'
 	Verify(data []byte, sig []byte) (bool, error)
-
-	// Encrypt data in a way that can be decrypted by a paired private key
-	Encrypt(data []byte) ([]byte, error)
```

Comment:
@Kubuxu is cryptobox what we want? https://godoc.org/golang.org/x/crypto/nacl/box#Seal